src/main/app/src/Components/Landscape/Modals/Group/Group.tsx

```diff
@@ -1,87 +1,104 @@
-import React, { useEffect, useState } from 'react';
-import { get } from '../../../../utils/API/APIClient';
-import './Group.scss';
+import React from 'react';
 
-import { IAssessmentProps, IGroup } from '../../../../interfaces';
-import { getLabels, getLinks, getGroupItems, getAssessmentSummary } from '../../Utils/utils';
+import { IAssessment, IGroup } from '../../../../interfaces';
+import { getLabels, getLinks, getAssessmentSummary, getItemIcon } from '../../Utils/utils';
+import { Card, CardHeader } from '@material-ui/core';
+import CardContent from '@material-ui/core/CardContent';
+import StatusChip from '../../../StatusChip/StatusChip';
+import componentStyles from '../../../../Ressources/styling/ComponentStyles';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Avatar from '@material-ui/core/Avatar';
 
 interface Props {
-  fullyQualifiedGroupIdentifier: string;
+  group: IGroup;
+  assessments: IAssessment;
   findItem?: (fullyQualifiedGroupIdentifier: string) => void;
   findGroup?: (fullyQualifiedGroupIdentifier: string) => void;
-  onAssessmentClick?: (fullyQualifiedItemIdentifier: string) => void;
 }
 
 /**
  * Returns a choosen Landscape group if informations are available
  */
-const Group: React.FC<Props> = ({
-  fullyQualifiedGroupIdentifier,
-  findItem,
-  findGroup,
-  onAssessmentClick,
-}) => {
-  const [group, setGroup] = useState<IGroup | undefined>();
-  const [assessment, setAssessment] = useState<IAssessmentProps[] | undefined>(undefined);
+const Group: React.FC<Props> = ({ group, assessments, findItem, findGroup }) => {
```

Comment:
I noticed that in the map, it is not possible to click on the group background to focus on that group. I would have expected to get a focus on it and to have the group information displayed at the "sidebar"

Reply:
https://github.com/dedica-team/nivio/issues/349